_data/current.yml

```diff
@@ -344,3 +344,10 @@
   url: https://stareast.techwell.com/speak-stareast?utm_source=testingconferences
   twitter: TechWell
   status: CFP is Open until September 10, 2023
+
+- name: Testμ 2024
+  location: Online
+  dates: "20 - 22 August, 2024"
+  url: https://www.lambdatest.com/testmuconf-2024?utm_source=STC&utm_medium=organic&utm_content=testmu_2024
+  twitter: lambdatesting
+  status: Registrations are Open | CFP is Open
```

Comment:
We like to list CFP first and then Registrations second. You can also use html and <a href> tags to provide links to both places in the status.